components/images/images-en.html

```diff
@@ -0,0 +1,249 @@
+
+<h3>Stretching images</h3>
+<p>Use to stretch a smaller image to the width of the container. Ensure the image is still clear and easy to see in the larger resolutions.</p>
+
+<h4>Appearance</h4>
+<div class="row">
+	<div class="col-md-4">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<p>Image (100x25)  is too small  and blurry: </p>
+				<img src="https://placehold.it/100x25" alt="Generic placeholder image">
+				<p class="mrgn-tp-md">Image (200x50) is still too small for the space:</p>
+				<img src="https://placehold.it/200x50" alt="Generic placeholder image">
+				<p class="mrgn-tp-lg">Stretched image (100x25) is blurry:</p>
+				<img src="https://placehold.it/100x25" class="full-width" alt="Generic placeholder image">
+				<p class="mrgn-tp-lg">Stretched, clear image (200x50)  fits space: </p>
+				<img src="https://placehold.it/200x50" class="full-width" alt="A generic square placeholder image">
+			</div>
+		</div>
+	</div>
+</div>
+
+<h4>Source code</h4>
+<pre><code>&lt;img src=&quot;https://placehold.it/200x50&quot; class=&quot;full-width&quot; alt=&quot;A generic square placeholder image&quot;&gt;</code></pre>
+
+<h3>Thumbnail tiles</h3>
+<p>Use to add any kind of content like headings, paragraphs, or buttons into a thumbnail, to create a thumbnail tile effect.</p>
+
+<h4>Appearance</h4>
+<div class="row">
+	<div class="col-md-4">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<figure class="thumbnail"><a href="#"><img src="https://placehold.it/350x200" alt="Generic placeholder thumbnail"></a>
+					<figcaption class="caption">
+						<h4>Title (caption) </h4>
+						<p>Content</p>
+						<ul class="list-inline">
+							<li><a href="#" class="btn btn-primary" role="button">Button</a></li>
+							<li><a href="#" class="btn btn-default" role="button">Button</a></li>
+						</ul>
+					</figcaption>
+				</figure>
+			</div>
+		</div>
+	</div>
+</div>
+
+<h4>Source code</h4>
+<pre><code>&lt;figure class=&quot;thumbnail&quot;&gt;&lt;a href=&quot;#&quot;&gt;&lt;img src=&quot;https://placehold.it/350x200&quot; alt=&quot;Generic placeholder thumbnail&quot;&gt;&lt;/a&gt;
+	&lt;figcaption class=&quot;caption&quot;&gt;
+		&lt;h4&gt;Title (caption) &lt;/h4&gt;
+		&lt;p&gt;Content&lt;/p&gt;
+		&lt;ul class=&quot;list-inline&quot;&gt;
+			&lt;li&gt;&lt;a href=&quot;#&quot; class=&quot;btn btn-primary&quot; role=&quot;button&quot;&gt;Button&lt;/a&gt;&lt;/li&gt;
+			&lt;li&gt;&lt;a href=&quot;#&quot; class=&quot;btn btn-default&quot; role=&quot;button&quot;&gt;Button&lt;/a&gt;&lt;/li&gt;
+		&lt;/ul&gt;
+	&lt;/figcaption&gt;
+&lt;/figure&gt;</code></pre>
+
+<h3>Media objects</h3>
+<p>Use to layer text and images for media content like blog comments, tweets, and so forth.</p>
+
+<h4>Appearance</h4>
+<div class="row">
+	<div class="col-md-4">
+		<div class="panel panel-default">
+			<div class="panel-body">
```

Comment:
This example is pretty tight, can we remove the panel and simply render it as 3 column, one per example?

Reply:
Kept as is for now